package.json

```diff
@@ -7,7 +7,8 @@
     "Juan Carlos Farah",
     "Hassan Abdul Ghaffar",
     "Mashkour Abdel Aziz",
-    "Kim Lan Phan Hoang"
+    "Kim Lan Phan Hoang",
+    "Hagop Taminian"
   ],
   "homepage": ".",
   "license": "MIT",
```

Comment:
I think we'll make this AGPLv3. Maybe change and add the updated license in another PR. Note that there is usually a special string for AGPLv3 in package.json. I think we've used: `"license": "AGPL-3.0-only"`

Reply:
you're right, according to https://spdx.org/licenses/ the identifier _AGPL-3.0_ is deprecated; it has to be either _AGPL-3.0-only_ or _AGPL-3.0-or-later_